package.json

```diff
@@ -117,7 +117,7 @@
     "babel-plugin-lodash": "3.3.4",
     "commander": "8.1.0",
     "copy-webpack-plugin": "9.0.1",
-    "cypress": "^10.9.0",
+    "cypress": "^12.13.0",
```

Comment:
not sure upgrading major versions is a good thing.. since it's generally preferred to stay consistent with other apps. if we do major deps upgrade,  we have to make them everywhere